main/lsp/requests/sorbet_show_symbol.cc

```diff
@@ -14,7 +14,11 @@ unique_ptr<ResponseMessage> SorbetShowSymbolTask::runRequest(LSPTypecheckerDeleg
     auto response = make_unique<ResponseMessage>("2.0", id, LSPMethod::SorbetShowSymbol);
 
     const core::GlobalState &gs = typechecker.state();
-    auto result = queryByLoc(typechecker, params->textDocument->uri, *params->position, LSPMethod::SorbetShowSymbol);
+    // To match the behavior of Go To Definition, we don't error in an untyped file, but instead
+    // return an empty result.
+    auto errorIfFileIsUntyped = false;
+    auto result = queryByLoc(typechecker, params->textDocument->uri, *params->position, LSPMethod::SorbetShowSymbol,
+                             errorIfFileIsUntyped);
```

Comment:
This comment is good, but it's not obvious to me exactly how the empty result is returned in untyped files (compare `DefinitionTask::runRequest`, which I guess is the implementation of Go To Definition: explicit checks for `fileIsTyped`).

Does this just happen magically below because `sym` won't exist for non-constant things?  If that's the case, maybe we should be a little more explicit in this comment that we can return non-empty results for some things, and consider adding a note below?

Reply:
Yeah it's maybe more straightforward to say "..., but instead be okay with returning an empty result"

constants and method defs will have a result, method calls will not.